src/gsy_e/models/strategy/heat_pump.py

```diff
@@ -80,7 +98,7 @@ def event_bid_traded(self, *, _market_id: str, bid_trade: Trade) -> None:
             return
 
         time_slot = bid_trade.time_slot
-        if bid_trade.buyer_id != self.owner.uuid:
+        if bid_trade.buyer.origin_uuid != self.owner.uuid:
```

Comment:
I do not necessarily mind for this one, since for the case of the Trade in the "Home" market the origin_uuid is the same as the "uuid", however I was wondering whether you stumbled upon this issue when running the simulation.